README.md

```diff
@@ -0,0 +1,17 @@
+## Meta information about a threat modeling activity
+
+<quote>
+Information identifying the threat model typically includes the following:
+1. Application Name: The name of the application examined.
+2. Application Version: The version of the application examined.
```

Comment:
Can the version be optional?

Reply:
Yes, of course. 
This was just a related block-quote from an OWASP article. 

Improved the README and stated our information requirements.